app/main.py

```diff
@@ -1,6 +1,28 @@
+import os
+import requests
+
+API_KEY = os.environ["API_KEY"]
+
+URL = (
+    f"http://api.openweathermap.org/data/2.5/weather"
+    f"?q=Paris"
+    f"&appid={API_KEY}"
+    f"&units=metric"
+)
```

Comment:
You've hardcoded the city name `Paris` into your URL. According to the checklist, you should use a constant for this. This makes your code more maintainable and easier to understand.